lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -137,9 +140,13 @@
                 }
             ];
 
-            if (_.isArray(authData.providers)) {
-                var accountsSectionData = {
+            var accountsSectionData = [
+                {
                     title: gettext('Connected Accounts'),
```

Comment:
Linked Accounts

(I've changed "connect/connected" to "link/linked" throughout for consistency.)